mimic/rest/maas_api.py

```diff
@@ -276,7 +277,15 @@ def create_multiplot_from_metric(metric, reqargs, allchecks):
                     d = {}
                     d['numPoints'] = 4
                     d['timestamp'] = timestamp
-                    d['average'] = random.randint(1, 99)
+                    if not q % (points / 4):
+                        squarewave_downtrend = squarewave_downtrend ^ 1
+                    if c['label'] == 'squarewave':
```

Comment:
This label is never "squarewave" in the tests, which means these lines of code are never tested.